services/wallet/thirdparty/market/coingecko/client.go

```diff
@@ -14,6 +12,8 @@ import (
 	"github.com/status-im/status-go/services/wallet/thirdparty/utils"
 )
 
+var baseURL = "https://api.coingecko.com/api/v3"
```

Comment:
having this as a member variable instead of a global was better (for example, you could have direct access and proxy-access to coingecko simply by having two instances pointing to different urls)